Cargo.toml

```diff
@@ -15,11 +15,13 @@ categories = ["graphics"]
 features = ["mint", "schemars", "serde"]
 
 [features]
-default = ["std"]
-std = []
+default = ["std", "alloc"]
```

Comment:
My brain is fried, but can't this just be `default = ["std"]` since the line below handles enabling `"alloc"`?